### DISCUSSION TITLE
Multi Module developing question

### DISCUSSION THREAD

**Discussion by Grieey** [fzyzcjy/flutter_rust_bridge#501] [Q&A] [answered]
Is there any solutions for multi module developing? The example shows there is only one bridge between flutter and rust, with the project growing, and the api.rs file will be large too, so splitting different feature into different module and reusing feature are both  difficult.

**Comment by fzyzcjy** [comment, marked as answer]
Maybe duplicate of https://github.com/fzyzcjy/flutter_rust_bridge/issues/495 ?

**Comment by dbsxdbsx** [comment]
> Is there any solutions for multi module developing? The example shows there is only one bridge between flutter and rust, with the project growing, and the api.rs file will be large too, so splitting different feature into different module and reusing feature are both difficult.

I guess you are looking for this: http://cjycode.com/flutter_rust_bridge/article/generate_multiple_files.html.
